src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -274,9 +274,9 @@ public partial struct JsonWriterOptions
         public bool TrySkip() { throw null; }
         public bool ValueTextEquals(System.ReadOnlySpan<byte> utf8Text) { throw null; }
         public bool ValueTextEquals(System.ReadOnlySpan<char> text) { throw null; }
-        public bool ValueTextEquals(string text) { throw null; }
+        public bool ValueTextEquals(string utf8Text) { throw null; }
```

Comment:
`text` is the correct parameter name (and the ref is correct) 

The source needs to change and be `text. `utf8Text` is not the right name.
https://github.com/dotnet/corefx/blob/df5bb8e4509ca5166f83abbd4cfe4a063433fee7/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L438

(also the xml comment/param name would need to be updated)

Reply:
I'll go ahead and change it then. Do note that the overload that takes a ReadOnlySpan of bytes param name is also `utf8Text`

Reply:
Yep, that is correct and expected. `ReadOnlySpan<byte>` contains utf-8 encoded text (think of it as a place holder for `Utf8String`, which is why want to specify utf-8 in the parameter name). `System.String` is **utf-16** encoded text.